qrisk/stats.py

```diff
@@ -290,18 +294,22 @@ def calmar_ratio(returns, period=DAILY):
     return temp
 
 
-def omega_ratio(returns, annual_return_threshhold=0.0):
+def omega_ratio(returns, mar=0.0, bdays=APPROX_BDAYS_PER_YEAR):
     """Determines the Omega ratio of a strategy.
 
     Parameters
     ----------
     returns : pd.Series
         Daily returns of the strategy, noncumulative.
-         - See full explanation in tears.create_full_tear_sheet.
-    annual_return_threshold : float, optional
-        Threshold over which to consider positive vs negative
-        returns. For the ratio, it will be converted to a daily return
-        and compared to returns.
+        - See full explanation in cum_returns.
+    mar : float, optional
```

Comment:
I would just call it `minimum_acceptance_return`, more descriptive always beats saving a few characters.